utils/atomic/funcs_test.go

```diff
@@ -0,0 +1,36 @@
+package atomic
+
+import (
+	"flag"
+	"sync/atomic"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"pgregory.net/rapid"
+)
+
+func TestUint64Sub(t *testing.T) {
+	_ = flag.Set("rapid.checks", "50")
+	_ = flag.Set("rapid.steps", "50")
+
+	rapid.Check(t, func(r *rapid.T) {
+		exp := uint64(0)
+		got := atomic.Uint64{}
+		r.Repeat(map[string]func(r *rapid.T){
+			"": func(r *rapid.T) {
+				require.Equal(t, exp, got.Load())
+			},
+			"add": func(r *rapid.T) {
```

Comment:
why is this case called "add" if you are testing a subtraction function? Is this a _rapid_ thing?

Reply:
no, you can add negative numbers

Reply:
changed anyway